proto/hummock.proto

```diff
@@ -246,6 +246,8 @@ message CompactTask {
   map<uint32, TableOption> table_options = 17;
   uint64 current_epoch_time = 18;
   uint64 target_sub_level_id = 19;
+  // compaction output table stats, which is 1-1 mapped to [`sorted_output_ssts`].
+  repeated TableStatsRef output_ssts_table_stats = 20;
```

Comment:
Shall we extract both sorted_output_ssts and output_ssts_table_stats out of CompactTask?